Examples/LambdaFunctions/scripts/package.sh

```diff
@@ -17,11 +17,15 @@ set -eu
 
 executable=$1
 
-target=.build/lambda/$executable
+target=".build/lambda/$executable"
 rm -rf "$target"
 mkdir -p "$target"
 cp ".build/release/$executable" "$target/"
-cp -Pv /usr/lib/swift/linux/lib*so* "$target"
+#cp -Pv /usr/lib/swift/linux/lib*so* "$target"
```

Comment:
leftover?

Reply:
yes, intentional for reference while we discuss the "new" way